charts/hivemq-platform-operator/values.yaml

```diff
@@ -42,6 +42,18 @@ serviceAccount:
   # The service account name to be used, or "hivemq-<release name>" if not set.
   name: ""
 
+# Configures the service account and permissions the operator creates for the HiveMQ Platform Pods.
+platformServiceAccount:
```

Comment:
I find this a bit confusing. So there are two service accounts, one for the operator to do its work and one for each hivemq platform to access the operator? We should improve the description for both.

I would suggest to find a different name than "platformServiceAccount". How about hivemqPodServiceAccount or hivemqPlatformServiceAccount?

Reply:
The ServiceAccount for the Platform is for the init app in the Platform Pods, so it can add annotations to the Pod itself.

The Platform is not accessing the operator via K8s. The Platform Pods execute HTTP requests to the operator service to download the init app, but that's a network policy, nothing in the operator deployment.

Reply:
I like `hivemqPlatformServiceAccount` better, that since we don't use "pod" in the naming of the HiveMQ Platform Helm chart.

Reply:
Renamed the `values.yaml` and all usages to `hivemqPlatformServiceAccount`. Also polished the descriptions a bit (and aligned how we spell K8s resource names).